src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -103,6 +103,19 @@ private void initialize() {
                     });
                     addToggle.getStyleClass().add("addEntryButton");
                     addToggle.selectedProperty().bindBidirectional(entriesListView.getItemBooleanProperty(entry));
+                    addToggle.selectedProperty().addListener((observable, oldValue, newValue) -> {
+                        if(observable.getValue()){
```

Comment:
Please follow our code formatting guidelines. The auto formatter should add some spaces here.